qcodes/instrument_drivers/Keysight/keysight_e4980a.py

```diff
@@ -236,6 +248,53 @@ def __init__(self,
                       "the auto range function OFF."
         )
 
```

Comment:
Now some (all?) of these parameters should only be added if has_option_001 is True right?

Reply:
No, these parameters are still there but with different allowed ranges. All the implemented parameters take ranges which are allowed with option 001.

Reply:
Ok that just doesn't match your original implementation nor the comment on @astafan8 s comment. 

If the ranges depend on the option they should be set according to the option so they are always correct.

Reply:
Yes, my original implementation was not correct. I had misunderstood how the options are making a difference. My comment 'Updated as discussed' reflects an offline discussion I had will @astafan8 regarding updating all parameters according to options.

Sure, I'll update the parameters accordingly.